flink-connectors/flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/enumerator/PulsarSourceEnumStateSerializer.java

```diff
@@ -54,57 +55,37 @@ private PulsarSourceEnumStateSerializer() {
 
     @Override
     public int getVersion() {
-        // We use PulsarPartitionSplitSerializer's version because we use reuse this class.
-        return PulsarPartitionSplitSerializer.CURRENT_VERSION;
+        return CURRENT_VERSION;
     }
 
     @Override
     public byte[] serialize(PulsarSourceEnumState obj) throws IOException {
-        // VERSION 0 serialization
         try (ByteArrayOutputStream baos = new ByteArrayOutputStream();
                 DataOutputStream out = new DataOutputStream(baos)) {
             serializeSet(
                     out, obj.getAppendedPartitions(), SPLIT_SERIALIZER::serializeTopicPartition);
-            serializeSet(
```

Comment:
Shall we serialize dummy states for downgrading from version 1 to version 0?

Reply:
I don't think we need support downgrading.